src/app/loginComponents/accounts/external/accounts.component.scss

```diff
@@ -2,44 +2,7 @@
 @import '/src/materialColorScheme.scss';
 @import '/src/styles.scss';
 
-// Resizing the logos to fit nicely in the card
-.account-logo {
-  width: 3.5em;
```

Comment:
This width was larger than the Angular default 40px and much larger than the 36px in the zeplin